src/js/course.ts

```diff
@@ -470,6 +485,16 @@ export function isClosed(c: CourseV3) {
   return c.courseEnrollmentStatus === "closed";
 }
 
+export function isTech(c: CourseV3) {
+  const code = c.courseCode.split(" ")[0];
+  return filterTechs.includes(code) && code !== "PE";
+}
+
+export function isHum(c: CourseV3) {
+  const code = c.courseCode.split(" ")[0];
+  return !filterTechs.includes(code) && code !== "PE";
+}
+
```

Comment:
Hmm, I think a more TypeScript-y, and also future-proof, way to implement this sort of categorization is via enums.  I suggest declaring an enum type (named `Area` or `Techiness` or something you can come up with that's better) with entries `Tech`, `Hum`, and `Pe`, and having a function `area(c: CourseV3): Area` instead of separate `isTech`, `isHum` functions and having to check `!== "PE"` in each of them.

Reply:
That's a great organizational idea. I will go ahead and do that soon!